server.js

```diff
@@ -2040,6 +2040,135 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub pullrequests integration.
+camp.route(/^\/github\/issues-pr(-raw)?\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Well, the idea was to combine all github issues support into a single function, because we can programmatically detect all the `issues<-something>` and send those that apply to github as `+is:<something>`. Is there a reason why you chose to duplicate all the code?